FrameworkSystem/DB/ProxyDB.py

```diff
@@ -534,11 +537,89 @@ def renewFromMyProxy( self, userDN, userGroup, lifeTime = False, chain = False )
     self.logAction( "myproxy renewal", hostDN, "host", userDN, userGroup )
     return S_OK( mpChain )
 
+  def __isPUSPdn( self, userDN ):
```

Comment:
In fact it so minor to make it a separate utility. isPUSP works with another object ( X509Chain ), here this is just a DN. I will drop it to avoid confusion